static/app/components/profiling/ProfilingOnboarding/profilingOnboardingModal.tsx

```diff
@@ -238,6 +246,44 @@ function SelectProjectStep({
   );
 }
 
+function usePublicDSN({
+  organization,
+  project,
+}: {
+  organization: Organization | null | undefined;
```

Comment:
why both null and undefined type for org and project?

Reply:
must have been left over from a prev revision as i was working on this, i'll fix it up